packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -780,23 +780,23 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
       final String bId = service.toId(elementB, group)!;
       final Object? jsonList = json.decode(service.getParentChain(bId, group));
       expect(jsonList, isList);
-      final List<Object> chainElements = jsonList! as List<Object>;
+      final List<dynamic> chainElements = jsonList! as List<dynamic>;
```

Comment:
Here and everywhere where `dynamic` is used: Why not `Object?`? I think that's what we've been using elsewhere for these cases.

Reply:
I've switched to using Object? everywhere.